tritonbench/operators/fp8_attention/operator.py

```diff
@@ -110,7 +110,7 @@ def triton_flash_v2(
         triton_q, triton_k, triton_v = self.triton_preprocess(q, k, v)
         # full fp8 will be enabled if type of q,k,v is fp8
         return lambda: triton_attention(
-            triton_q, triton_k, triton_v, False, self.sm_scale
+            triton_q, triton_k, triton_v, False, self.sm_scale, "base"
```

Comment:
cc @manman-ren `attention_opt` will compile error on the pytorch version of Triton, does it require the latest Triton main branch?

Reply:
Sorry for the breakage. What is the error message?

Reply:
@manman-ren here is the error message: https://github.com/pytorch-labs/tritonbench/actions/runs/11903695593/job/33171153429?pr=55. By default we are using the pytorch built-in Triton in the CI.